front/src/components/ViewMockup.js

```diff
@@ -49,19 +64,61 @@ class ViewMockup extends Component {
         window.location.href = "/mockupsurvey/add/" + this.state.mockup.id
     };
 
+    startButton() {
+        let displayMediaOptions = {
+            video: {
+                cursor: "always"
+            },
+            audio: false
+        }
+        let video = document.getElementById("video")
+        navigator.mediaDevices.getDisplayMedia(displayMediaOptions)
+            .then(stream => {
+                video.srcObject = stream
+                return new Promise(resolve => video.onplaying = resolve)
+            })
+            .then(() => {
+                let startButton = document.getElementById("startButton")
+                let stopButton = document.getElementById("stopButton")
+                startButton.hidden = true;
+                stopButton.hidden = false;
+                return startRecording(video.captureStream())
+            })
+            .then(recordedChunks => {
+                let a = document.createElement('a');
+                a.href = window.URL.createObjectURL(new Blob(recordedChunks, {type: "video/webm"}));
+                a.download = "Recording.webm";
+                a.click();
+            })
+    }
+
+    stopButton() {
+        let video = document.getElementById("video")
+        video.srcObject.getTracks().forEach(track => track.stop())
+        let startButton = document.getElementById("startButton")
+        let stopButton = document.getElementById("stopButton")
+        startButton.hidden = false;
+        stopButton.hidden = true;
+    }
+
     render() {
         if (!AuthenticationService.isUserLoggedIn()) {
             return <Redirect to={"/"}/>
         }
         return (
             <div className="bg-light">
+                <video id="video" hidden autoPlay/>
                 <nav className="navbar navbar-expand-lg navbar-light navbar-secondary">
                     <span className="container float-left navbar-breadcrumbs">
                         <a href="/project">Projects</a> &ensp; / &ensp;
                         <a href={"/project/view/" + this.state.mockup.projectId}>{this.state.projectName}</a> &ensp; / &ensp;
                         {this.state.mockup.name}
                     </span>
                     <span className="container float-right navbar-buttons">
+                        <button className="btn btn-primary" id="startButton"
```

Comment:
`startRecordingButton` would be better than `startButton`. the same with `stopButton` and all variables corresponding